docs/dasharo-tools-suite/documentation/features.md

```diff
@@ -561,24 +327,24 @@ If you used `flashrom` as described in [Local firmware
 update](#local-firmware-update), flash the firmware again, but make sure the
 BIOS lock is turned off this time.
 
-### EC transition
+## EC transition
 
 DTS allows performing full Embedded Controller firmware transition from the
 proprietary vendor EC firmware to the Dasharo EC firmware. Currently, this
 functionality is supported on:
 
-* [NovaCustom NS5x/NS7x](../variants/novacustom_ns5x_tgl/releases.md),
-* [NovaCustom NV4x](../variants/novacustom_nv4x_tgl/releases.md),
-* [NovaCustom V540TU](../variants/novacustom_v540tu/releases.md),
-* [NovaCustom V540TNx](../variants/novacustom_v540tnx/releases.md),
-* [NovaCustom V560TU](../variants/novacustom_v560tu/releases.md)
-* [NovaCustom V560TNx](../variants/novacustom_v560tnx/releases.md).
+* [NovaCustom NS5x/NS7x](../../variants/novacustom_ns5x_tgl/releases.md),
+* [NovaCustom NV4x](../../variants/novacustom_nv4x_tgl/releases.md),
+* [NovaCustom V540TU](../../variants/novacustom_v540tu/releases.md),
+* [NovaCustom V540TNx](../../variants/novacustom_v540tnx/releases.md),
+* [NovaCustom V560TU](../../variants/novacustom_v560tu/releases.md)
+* [NovaCustom V560TNx](../../variants/novacustom_v560tnx/releases.md).
 
 Starting from DTS v1.2.0 to perform EC transition please run
 [firmware update](#firmware-update) on the platform with proprietary vendor EC
 firmware.
 
-### EC update
+## EC update
```

Comment:
Could you fix indent:

````diff
* Retrieve information about your current EC.

-  ```bash
-  dasharo_ectool info
-  ```

-  The output of the above-described command should contain information about
-  the version of flashed firmware:

-  ```bash
-  board: clevo/ns50mu
-  version: 2022-08-16_c12ff1a
-  ```
+    ```bash
+    dasharo_ectool info
+    ```

+    The output of the above-described command should contain information about
+    the version of flashed firmware:

+    ```bash
+    board: clevo/ns50mu
+    version: 2022-08-16_c12ff1a
+    ```

* Download the newest version of Embedded Controller firmware.
* Plug in power supply, without it, flashing EC is not possible as losing power
  may cause in damaged firmware.
* Flash Embedded Controller firmware internally.

-  ```bash
-  dasharo_ectool flash ec_file.rom
-  ```

-  The output of the above-described command should look as follows:

-  ```bash
-  file board: Ok("clevo/ns50mu")
-  file version: Ok("2022-08-16_c12ff1a")
-  ec board: Ok("clevo/ns50mu")
-  ec version: Ok("2022-08-31_cbff21b")
-  Waiting 5 seconds for all keys to be released
-  Sync
-  SPI Read 128K
-  Saving ROM to backup.rom
-  SPI Write 128K
-  SPI Read 128K
-  Successfully programmed SPI ROM
-  Result: Ok(())
-  Sync
-  System will shut off in 5 seconds
-  Sync
-  ```

-  > Note: this is example output, versions may differ

+    ```bash
+    dasharo_ectool flash ec_file.rom
+    ```

+    The output of the above-described command should look as follows:

+    ```bash
+    file board: Ok("clevo/ns50mu")
+    file version: Ok("2022-08-16_c12ff1a")
+    ec board: Ok("clevo/ns50mu")
+    ec version: Ok("2022-08-31_cbff21b")
+    Waiting 5 seconds for all keys to be released
+    Sync
+    SPI Read 128K
+    Saving ROM to backup.rom
+    SPI Write 128K
+    SPI Read 128K
+    Successfully programmed SPI ROM
+    Result: Ok(())
+    Sync
+    System will shut off in 5 seconds
+    Sync
+    ```

+    > Note: this is example output, versions may differ

* Computer will shut down automatically.
* Power on your computer. Booting process may take a while.
* After boot, choose option `S` to drop to Shell.
* Retrieve information about your updated EC.

-  ```bash
-  dasharo_ectool info
-  ```

-  The output of the above-described command should contain information about
-  the version of flashed firmware:

-  ```bash
-  board: clevo/ns50mu
-  version: 2022-08-31_cbff21b
-  ```
+    ```bash
+    dasharo_ectool info
+    ```

+    The output of the above-described command should contain information about
+    the version of flashed firmware:

+    ```bash
+    board: clevo/ns50mu
+    version: 2022-08-31_cbff21b
+    ```
````

Reply:
If you looked at the diff I posted you would see I added 2 spaces to make it proper indent:

Without indent:

![image](https://github.com/user-attachments/assets/358246ad-a33e-4211-8a95-39a87707add7)

With indent:

![image](https://github.com/user-attachments/assets/8139caaf-1f00-48bd-8c2c-879a2528fff2)

Reply:
Ah, I build it and checked once again and now I see. Mkdocs wants one more level of indent for it to be displayed properly.

Reply:
90f9fd4c503a04a984a9016614ed7cef2c6c505a

Reply:
It probably depends but I saw that a lot of times markdown viewers don't accept 2 spaces as indent by default (unless configured somehow).

But printscreen wasn't made on GitHub but on local website run by using `mkdocs serve`